plugins/modules/zos_unarchive.py

```diff
@@ -35,7 +35,7 @@
       - The remote absolute path or data set of the archive to be uncompressed.
       - I(src) can be a USS file or MVS data set name.
       - USS file paths should be absolute paths.
-      - MVS data sets supported types are C(SEQ), C(PDS), C(PDSE).
```

Comment:
Should line 38 be removed?  I thought you added gdg, but didn't remove SEQ/PDS/et cetera.

Reply:
I'm wondering the same